### PR TITLE
Add permission to bypass country check

### PR DESCRIPTION
**Pull request by sgdc3**
#1321 Still need to fix unit tests

### CODE REVIEW

**Comment by sgdc3** [comment]
Any idea how to fix that damn mokito issue? @ljacqu

File: src/main/java/fr/xephi/authme/permission/PlayerStatePermission.java [hunk 1]
```diff
@@ -29,7 +29,12 @@
     /**
      * Permission to bypass the purging process.
      */
-    BYPASS_PURGE("authme.bypasspurge", DefaultPermission.NOT_ALLOWED);
+    BYPASS_PURGE("authme.bypasspurge", DefaultPermission.NOT_ALLOWED),
+
+    /**
+     * Permission to use to see own other accounts.
```
Comment (ljacqu) [inline src/main/java/fr/xephi/authme/permission/PlayerStatePermission.java]: Please update the sentence

File: src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java [hunk 2]
```diff
@@ -499,6 +505,12 @@ private static Player newPlayerWithName(String name) {
         return player;
     }
 
+    private static Player newPlayerWithAddress(String ip) {
```
Comment (ljacqu) [inline src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: Please use the method in TestUtils instead

File: src/main/java/fr/xephi/authme/permission/PlayerStatePermission.java (hunk 1 above)
Comment (sgdc3) [reply src/main/java/fr/xephi/authme/permission/PlayerStatePermission.java]: whoops, right

File: src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java (hunk 2 above)
Comment (sgdc3) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: can't find that method :/

Comment (ljacqu) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: TestHelper#mockPlayerIp

Comment (sgdc3) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: Ok fixing that, test are failing due to somereflection issues with BukkitTasks :/ could you take a look ? Probably something changed in bukkit.

Comment (ljacqu) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: Not atm, I'm abroad for the upcoming three weeks

Comment (ljacqu) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: Not atm, I'm abroad for the next three weeks

Comment (sgdc3) [reply src/test/java/fr/xephi/authme/listener/OnJoinVerifierTest.java]: Ok, i'll try to fix them

**Pull request merged**